eng/pipelines/code-quality-reports.yml

```diff
@@ -1,6 +1,8 @@
 jobs:
   - job: 'Linting'
 
+    timeoutInMinutes: 120
```

Comment:
Do we need to wait 2 hours for this pipeline to complete? Is it possible to break this down into 3 parallel stages - one for data plane, one for Spring and one for Management libraries to speed this up?

Reply:
Right now, it takes about 80 minutes for Checkstyle, RevApi, and Spotbugs validation, but all three won't commonly run at the same time. For now, I think this is an acceptable PR wait time for something that affects the entire repo. If it become problematic we can split this into parallel runs but I'd split it into whole repo runs split on Checkstyle, RevApi, and Spotbugs so that the base logic in this PR can be retain to no-op jobs if they aren't required.